app.js

```diff
@@ -65,9 +64,6 @@ app.use(handleRender);
 app.use('/api/user', user);
 app.use('/api/session', userSession);
 
-// Setup authentication for routes after
-app.use(handleAuthentication);
-
```

Comment:
router specific files should use this middleware if they require auth. This should not be set globally